applications/search/src/pages/search-page/results-informationmodel/results-informationmodel.component.jsx

```diff
@@ -138,6 +121,16 @@ export const ResultsInformationModelPure = ({
           <span className="uu-invisible" aria-hidden="false">
             Filtrering
           </span>
+
+          <FilterPills
+            history={history}
+            location={location}
+            locationSearch={locationSearch}
+            publishers={publishers}
+            handleClearFilters={onClearFilters}
+            showClearFilterButton={showClearFilterButton}
```

Comment:
Jeg tenkte, at grunnen til å bevege "clearFilters" knapp inni FilterPills er, at alt lignende filter manipulasjon logikk er nå i FilterPills. Så at FilterPills skal bestemme, hvis å rendre clear all knapp, og også, hvordan å håndtere clear (ved help av. 
Så vi skulle unngå de to props : handleClearFilters, showClearFilterButton

Det kan godt være en ant pr, "Move clear filter handling to FilterPills component", men samtidlig, den nåværende PR kan også ses som extension av ClearFilters - "Extend clear filters button with filter pills block".

Reply:
Gjort